integration/token/fungible/tests.go

```diff
@@ -730,6 +741,44 @@ func TransferCash(network *integration.Infrastructure, id string, wallet string,
 	}
 }
 
+func PrepareTransferCash(network *integration.Infrastructure, id string, wallet string, typ string, amount uint64, receiver string, auditor string, tokenID *token2.ID, errorMsgs ...string) []byte {
+	txBoxed, err := network.Client(id).CallView("prepareTransfer", common.JSONMarshall(&views.Transfer{
+		Auditor:   auditor,
+		Wallet:    wallet,
+		TokenIDs:  []*token2.ID{tokenID},
+		Type:      typ,
+		Amount:    amount,
+		Recipient: network.Identity(receiver),
+	}))
+	if len(errorMsgs) == 0 {
+		Expect(err).NotTo(HaveOccurred())
+	} else {
+		Expect(err).To(HaveOccurred())
+		for _, msg := range errorMsgs {
+			Expect(err.Error()).To(ContainSubstring(msg))
+		}
+		time.Sleep(5 * time.Second)
```

Comment:
why the sleep?

Reply:
I don't remember to be honest. I see it also in the other functions. I can submit another PR where I remove all these sleeps and see what happens. Wdyt?